index.js

```diff
@@ -1,28 +1,35 @@
 const express = require('express');
-const bodyParser = require('body-parser')
+const bodyParser = require('body-parser');
 const exphbs = require('express-handlebars');
+const mongoose = require('mongoose');
+require('dotenv').config();
 
-const leaderboardRouter = require('./routers/leaderboardRouter');
+const leaderboardApi = require('./routers/leaderboardApi');
 
 const PORT = process.env.PORT || 3000
 
 const app = express();
+mongoose.connect(process.env.MONGO, { useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true });
+mongoose.Promise = global.Promise;
 
 app.engine('hbs', exphbs({
     defaultLayout: 'main',
     extname: '.hbs'
 }));
 // Setting template Engine
 app.set('view engine', 'hbs');
-app.use('/static', express.static(__dirname + '/public'))
+app.use('/public', express.static(__dirname + '/public'))
```

Comment:
Recommend to use `path.join` or `path.resolve`.

Reply:
Solved